packages/components/src/Enumeration/Items/Item/Item.test.js

```diff
@@ -64,6 +64,24 @@ describe('Item', () => {
 		expect(props.item.itemProps.onSelectItem).toBeCalled();
 	});
 
+	it('should display value with only button which are not disabled', () => {
+		// given
+		const itemWithDisabled = { ...item };
+		itemWithDisabled.itemProps.actions[0].disabled = true;
+		const props = {
+			item: itemWithDisabled,
+		};
+
+		const itemInstance = <Item {...props} />;
+
+		// when
+		const wrapper = mount(itemInstance);
+		const buttons = wrapper.find(Button);
+
+		// then
+		expect(buttons.length).toBe(2);
```

Comment:
I don't understand the assertion. You have 2 actions, 1 disabled, tou should have only 1 button

Reply:
the selection also includes the title which is clickable and is indeed a button, i've narrow the selection so we only check the one we're interested in.